Source/Core/InputCommon/ControllerInterface/XInput/XInput.h

```diff
@@ -90,7 +90,7 @@ class Device : public Core::Device
 
 private:
 	XINPUT_STATE m_state_in;
-	XINPUT_VIBRATION m_state_out;
+	XINPUT_VIBRATION m_state_out, m_current_state_out;
```

Comment:
I would prefer if somebody else would do it. I don't know for sure if the constructor might be called 2x, like once per game you start. And XINPUT_STATE m_state_in should also get the same treatment, which is out of scope for the pr.